app/lib/cubits/current_index.dart

```diff
@@ -640,21 +640,46 @@ class CurrentIndexCubit extends Cubit<CurrentIndex> {
       if (area == null || page == null) {
         continue;
       }
-      final pageFormat =
-          PdfPageFormat(area.width * quality, area.height * quality);
+
       final image = await render(document, page, info,
           width: area.width,
           height: area.height,
           x: area.position.x,
           y: area.position.y,
           quality: quality,
           renderBackground: renderBackground);
-      if (image == null) continue;
-      pdf.addPage(pw.Page(
-          pageFormat: pageFormat,
-          build: (context) {
-            return pw.Image(pw.MemoryImage(image.buffer.asUint8List()));
-          }));
+      
+      // * * use this variable for set landscape
+      bool landscape = false;
+      if (landscape == true) {
```

Comment:
There are currently also the changes you made for pdf. Can you remove this?

Reply:
Sure. They are there by mistake